CMakeLists.txt

```diff
@@ -104,14 +104,19 @@ endif()
 # don't be fooled).
 add_subdirectory(third-party)
 add_subdirectory(base)
-add_subdirectory(compiler)
-add_subdirectory(core)
-add_subdirectory(comm-libs)
-if(THEROCK_ENABLE_MATH_LIBS)
-  add_subdirectory(math-libs)
-endif()
-if(THEROCK_ENABLE_ML_FRAMEWORKS)
-  add_subdirectory(ml-frameworks)
+
+if(NOT WIN32)
+  add_subdirectory(compiler)
+  add_subdirectory(core)
+  add_subdirectory(comm-libs)
+  if(THEROCK_ENABLE_MATH_LIBS)
+    add_subdirectory(math-libs)
+  endif()
+  if(THEROCK_ENABLE_ML_FRAMEWORKS)
+    add_subdirectory(ml-frameworks)
+  endif()
+else()
+  # TODO(#36): Enable more project builds on Windows and/or make the full list configurable.
 endif()
```

Comment:
These changes may be leading to confusion on https://github.com/nod-ai/TheRock/issues/36#issuecomment-2632149725. Might need a different approach.